release_images.yml

```diff
@@ -280,3 +280,35 @@ release_images:
       example: True
       disable_sm_tag: False  # [Default: False] This option is not used by Example images
       force_release: False
+  16:
+    framework: "mxnet"
+    version: "1.8.0"
+    training:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py37"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu110"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+                            # to images being published.
+      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+                            # has already been published. Re-released image will have minor version incremented by 1.
+    inference:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py37"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu110"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  17:
+    framework: "mxnet"
+    version: "1.8.0"
+    training:
+      device_types: ["gpu"]
+      python_versions: ["py37"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu110"
+      example: True
+      disable_sm_tag: False  # [Default: False] This option is not used by Example images
+      force_release: False
```

Comment:
We generally don't add images to this file until the RCs have been prepared at least once.

Reply:
Will remove.